eox_nelp/processors/tests/xapi/mixins.py

```diff
@@ -0,0 +1,57 @@
+"""This is a mixin file that allows to reuse test cases.
+
+Classes:
+    BaseCourseObjectTestCaseMixin: Test mixin for children classes of BaseCourseObjectMixin.
+"""
+from mock import patch
+from tincan import ActivityDefinition, LanguageMap
+
+from eox_nelp.processors.xapi import constants as eox_nelp_constants
+
+
+class BaseCourseObjectTestCaseMixin:
+    """This is a test mixin for classes which implements the BaseCourseObjectMixin mixin."""
+
+    def tearDown(self):  # pylint: disable=invalid-name
+        """Reset mocks"""
+        self.transformer_class.get_data.reset_mock()
+        self.transformer_class.get_object_iri.reset_mock()
+
+    @patch("eox_nelp.processors.xapi.mixins.get_course_from_id")
+    def test_get_object_method(self, get_course_mock):
```

Comment:
So now the mixin is tested in a general way.

Reply:
There are two file on tests_mixins that tests the mixin behavior of processors/xapi/mixins file  the other file is this one test/xapi/mixins , that contains mixins for tests cases